### PR TITLE
[AT-4336] Removing create/update Project, Workflows, and jobs

### PR DESCRIPTION
**Pull request by andher1802**
(insert scope and description here)

Items:
* [ ] Ran test & live-tests
* [ ] Implemented (new) unit tests
* [ ] Removed credentials
* [ ] Removed argument pointing to staging
* [ ] Updated [documentation](sdk.up42.com)

For release:
* [ ] Bumped version
* [ ] Added changelog
* [ ] Updated announcement banner

### CODE REVIEW

**Comment by dustgalactic** [comment]
@andher1802 Is it failing because of my commits? I'm not sure what to fix to be honest...

**Comment by andher1802** [comment]
> @andher1802 Is it failing because of my commits? I'm not sure what to fix to be honest...

I think it is failing because the pylint, It needs a new line at the end of every file and `workflow-reference.md` did not have it. I updated it.

**Review by dustgalactic — APPROVED** [review]

File: up42/job.py [hunk 1]
```diff
@@ -90,63 +88,6 @@ def is_succeeded(self) -> bool:
         """
         return self.status == "SUCCEEDED"
 
-    def track_status(self, report_time: int = 30) -> str:
```
Comment (javidq) [inline up42/job.py:93]: This is not create/edit functionality, why is it dropped?

File: up42/project.py [hunk 2]
```diff
@@ -203,53 +155,3 @@ def get_project_settings(self) -> List[Dict[str, str]]:
         response_json = self.auth._request(request_type="GET", url=url)
         project_settings = response_json["data"]
         return project_settings
-
-    @property
-    def max_concurrent_jobs(self) -> int:
```
Comment (javidq) [inline up42/project.py:208]: This is not create/edit functionality

File: up42/workflow.py [hunk 3]
```diff
@@ -859,14 +266,3 @@ def delete(self) -> None:
         self.auth._request(request_type="DELETE", url=url, return_text=False)
         logger.info(f"Successfully deleted workflow: {self.workflow_id}")
         del self
-
-    @property
-    def max_concurrent_jobs(self) -> int:
```
Comment (javidq) [inline up42/workflow.py:864]: this is not create/edit functionality

**Pull request merged**